### PR TITLE
Add schema.yml for endpoints. (Issue: MUR-10357)

### PR DESCRIPTION
**Pull request by lenaw**

### CODE REVIEW

File: endpoints/schema.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+paths:
```
Comment (sauvainr) [inline endpoints/schema.yml]: How about reusable parameters?

Comment (lenaw) [reply endpoints/schema.yml]: i will add it.

File: endpoints/schema.yml [hunk 2]
```diff
@@ -6,88 +6,77 @@ paths:
           in: query
           description: tags to filter by
           required: false
-          type: array
-          items:
-            type: string
-          collectionFormat: csv
+          type: string
```
Comment (sauvainr) [inline endpoints/schema.yml]: Why the change?

Comment (lenaw) [reply endpoints/schema.yml]: two reasons:
1. our swagger ui doesn't support all properties, i think keep the basic parameters is enough.
2. even us don't use properties like `collectionFormat`, i was trying to not give users too many properties to learn. just keep the simple or required ones.

we can add more properties though, but it will not be a simple START example for users.

Comment (sauvainr) [reply endpoints/schema.yml]: ok

File: endpoints/schema.yml (hunk 1 above)
Comment (lenaw) [reply endpoints/schema.yml]: added `parameters` in schema.yml file

File: endpoints/schema.yml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    in: path
+    description: ID of pet to fetch
+    required: true
+    type: integer
+    format: int64
+
+definitions:
+  Pet:
+    type: object
+    allOf:
```
Comment (sauvainr) [inline endpoints/schema.yml]: @lenaw I am not sure if we should document complex tags as this one, as it is not supported by swagger.

**Comment by sauvainr** [comment]
Released
